pkg/controller/job/controller_test.go

```diff
@@ -155,7 +155,7 @@ func TestControllerSyncJob(t *testing.T) {
 		"too few active pods, with controller error": {
 			2, 5,
 			fmt.Errorf("Fake error"), 1, 1, 0,
-			0, 0, 1, 1, 0, false,
+			1, 0, 1, 1, 0, false,
```

Comment:
I modified the FakePodControl to increase the creation counter for the failed creation (i.e., receiving the "Fake error") as well. See [here](https://github.com/kubernetes/kubernetes/pull/27600/files/07b9ded913cc9d1c92135c369bf563663a7f76da..753a363c8c78d8be7ea028b34036f514dde929bf#diff-44bba2c68a6ea2b4ca0dbb7f4465828eR535), I need this behavior in my tests.